src/main/scala/ai/privado/languageEngine/java/tagger/PrivadoTagger.scala

```diff
@@ -84,7 +84,7 @@ class PrivadoTagger(cpg: Cpg) extends PrivadoBaseTagger {
     println(s"${Calendar.getInstance().getTime} - --APITagger invoked...")
     rules.sinks
       .filter(rule => rule.nodeType.equals(NodeType.API))
-      .foreach(rule => apiTagger.setRuleAndApply(rule))
+      .foreach(rule => apiTagger.createAndApply())
```

Comment:
line 85 to 87 we can write `new APITagger().createAndApply()` as we are already running individual rule in generateOnPart function inside the Pass